src/site/content/en/blog/speedy-css-tip-animated-loader/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Speedy CSS Tip! Animated Loader
+subhead: Let's make an animated CSS loader with scoped custom properties and animation-timing-function
+date: 2022-09-22
+hero: image/Dyx9FwYgMyNqy1kMGx8Orz6q0qC3/vzvc5Ep1blYUQMYYnAKT.jpg
+authors:
+  - jheyy
+tags: 
+  - blog
+  - css
+---
+Hop over to [CodePen](https://pen.new) and create a new pen.
+
+Create the markup for our loader. Note the use of inline custom properties!
+
+```html
+<div class="loader" style="--count: 10">
+  <span style="--index: 0"></span>
+  <span style="--index: 1"></span>
+  <span style="--index: 2"></span>
+  <span style="--index: 3"></span>
+  <span style="--index: 4"></span>
+  <span style="--index: 5"></span>
+  <span style="--index: 6"></span>
+  <span style="--index: 7"></span>
+  <span style="--index: 8"></span>
+  <span style="--index: 9"></span>
+</div>
+
+```
+
+Bonus points for using a generator(Pug 👇) to configure the number of lines.
+
+```pug
+- const COUNT = 10
+.loader(style=`--count: ${COUNT}`)
+  - let i = 0
+  while i < COUNT
+    span(style=`--index: ${i}`)
+    - i++
+```
+
+Give our loader some styles.
+
+```css
+loader {
+  --size: 10vmin;
```

Comment:
```suggestion
  --size: 10vmin;

```

Reply:
I've seen this enforced by stylelint in the past. I think it helps with readability to put variable declarations at the top of a CSS rule and then separate the following rules with a line.